emacs.fnl

```diff
@@ -37,10 +37,14 @@
                      " -e '(spacehammer/edit-with-emacs "
                      pid " " title " " screen " )' &")
         co          (coroutine.create (fn [run-str]
-                                        (io.popen run-str)))]
-    ;; select all + copy
-    (hs.eventtap.keyStroke [:cmd] :a)
-    (hs.eventtap.keyStroke [:cmd] :c)
+                                        (io.popen run-str)))
+        prev        (hs.pasteboard.changeCount)
```

Comment:
I'm guessing this workaround is working? What happens if I select text, edit in emacs, then while it's selected I edit it again? Will it still work?

Reply:
If you initiate editing while you're already have another edit-with-emacs session, you'd lose the text in edit-with-emacs buffer. I don't think it's a big deal. At some point in future we may choose to address that.